addOns/commonlib/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add info URL.
 - Constant for default number of threads.
+- Added timing utils for fixing FPs
```

Comment:
This one needs to be moved to "Unreleased", can also be something like `Add utilities for time-based checks.`